src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/DialogTransmissions/Queries/Search/SearchDialogTransmissionDto.cs

```diff
@@ -0,0 +1,55 @@
+using Digdir.Domain.Dialogporten.Application.Features.V1.Common.Content;
+using Digdir.Domain.Dialogporten.Application.Features.V1.Common.Localizations;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities.Actors;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities.Attachments;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities.Transmissions;
+
+namespace Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.DialogTransmissions.Queries.Search;
+
+public sealed class SearchDialogTransmissionDto
+{
+    public Guid Id { get; set; }
+    public DateTimeOffset CreatedAt { get; set; }
+    public string? AuthorizationAttribute { get; set; }
+    public bool IsAuthorized { get; set; }
+    public string? ExtendedType { get; set; }
+    public Guid? RelatedTransmissionId { get; set; }
+    public DateTimeOffset? DeletedAt { get; set; }
+
+    public DialogTransmissionType.Values Type { get; set; }
+
+    public SearchDialogTransmissionActorDto Sender { get; set; } = null!;
+
+    public SearchDialogTransmissionContentDto Content { get; set; } = null!;
+    public List<SearchDialogTransmissionAttachmentDto> Attachments { get; set; } = [];
+}
+
+public sealed class SearchDialogTransmissionActorDto
+{
+    public DialogActorType.Values ActorType { get; set; }
+    public string ActorName { get; set; } = null!;
+    public string ActorId { get; set; } = null!;
```

Comment:
Id here too?

Reply:
Yep, I'll do it in the Actor model change PR